State_manager_code/StateManager.py

```diff
@@ -61,6 +63,7 @@ def __init__(self):
                                    States.TrialCompleted: [Events.LastTrial.value],
                                    States.TrialAbort: [Events.LastTrial.value,
                                                        Events.Mouse1InCenter.value + Events.Mouse2InCenter.value],
+                                   States.DecisionAbort: [Events.Mouse1InCenter.value + Events.Mouse2InCenter.value],
```

Comment:
Is this coming from Ben? Is the desired behavior is to wait until the mice return to center? If this is the plan, then you should also eliminate the timeout transition from DecitionAbort back to TrialStarted.

Reply:
yes ben oversaw the experimnent yesterday and asked for it,ill eliminate the timeout transition

Reply:
Please fix on this Branch. So that we can track and control the changes.